src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -224,15 +225,13 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
     selectPortfolioAccountMetadataByAccountId(state, poolAssetAccountMetadataFilter),
   )
 
-  const { data: poolAssetAccountAddress } = useQuery({
-    ...reactQueries.common.thorchainFromAddress({
-      accountId: poolAssetAccountId,
-      assetId: poolAsset?.assetId!,
-      opportunityId: activeOpportunityId,
-      wallet: wallet!,
-      accountMetadata: poolAssetAccountMetadata!,
-      getPosition: getThorchainLpPosition,
-    }),
+  const { data: poolAssetAccountAddress } = useThorchainFromAddress({
+    accountId: poolAssetAccountId,
+    assetId: poolAsset?.assetId!,
+    opportunityId: activeOpportunityId,
+    wallet: wallet!,
+    accountMetadata: poolAssetAccountMetadata!,
+    getPosition: getThorchainLpPosition,
     enabled: Boolean(poolAsset?.assetId && wallet && poolAssetAccountMetadata),
```

Comment:
can remove enabled here since the args are validated in hook now and would suggest the hook handles undefined gracefully so we don't need to include `!`